tests/unit/src/test/scala/tests/markdown/ErrorSuite.scala

```diff
@@ -32,25 +34,8 @@ class ErrorSuite extends BaseMarkdownSuite {
        |x + y + z
        |^^^^^^^^^
        |scala.NotImplementedError: an implementation is missing
-       |	at scala.Predef$.$qmark$qmark$qmark(Predef.scala:288)
-       |""".stripMargin,
-    compat = Map(
-      Compat.Scala213 ->
-        """|error: crash.md:10:1: an implementation is missing
-           |x + y + z
-           |^^^^^^^^^
-           |scala.NotImplementedError: an implementation is missing
-           |<TAB>at scala.Predef$.$qmark$qmark$qmark(Predef.scala:344)
-           |""".stripMargin.replace("<TAB>", tab),
-      Compat.Scala3 ->
-        // Anton: for some reason, the compiler(?) returns tab in this particular place
-        """|error: crash.md:10:1: an implementation is missing
-           |x + y + z
-           |^^^^^^^^^
-           |scala.NotImplementedError: an implementation is missing
-           |<TAB>at scala.Predef$.$qmark$qmark$qmark(Predef.scala:345)
-           """.stripMargin.replace("<TAB>", tab)
-    )
+       |	at scala.Predef$.$qmark$qmark$qmark(Predef.scala:???)
```

Comment:
Seems this was a TAB in each version. I also changed Predef lines to `???` since it seems this changes very often.